security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -6,7 +6,29 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates.
+The {cert-manager-operator} supports using Automated Certificate Management Environment (ACME) CA servers, such as Let's Encrypt, to issue certificates. Explicit credentials are configured by specifying the secret details in the `Issuer` API object. Ambient credentials are extracted from the environment, metadata services, or local files which are not explicitly configured in the `Issuer` API object.
+
+[NOTE]
+====
+`Issuer` is namespace scoped. It can only issue certificates from the same namespace. You can also use `ClusterIssuer` to issue certificates across all namespaces in the cluster.
+
+.Example YAML file that defines the `ClusterIssuer` object
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: acme-cluster-issuer
+spec:
+  acme:
+    ...
+----
+====
+
+[NOTE]
+====
+By default, you can use `ClusterIssuer` with ambient credentials. To use `Issuer` with ambient credentials, you must enable `--issuer-ambient-credentials` for the cert-manager controller.
+====
```

Comment:
I know it makes it wordy, but we should use nouns for all of these `monospaced` items
```suggestion
====
The `Issuer` object is namespace scoped. It can only issue certificates from the same namespace. You can also use the `ClusterIssuer` object to issue certificates across all namespaces in the cluster.

.Example YAML file that defines the `ClusterIssuer` object
[source,yaml]
----
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: acme-cluster-issuer
spec:
  acme:
    ...
----
====

[NOTE]
====
By default, you can use the `ClusterIssuer` object with ambient credentials. To use the `Issuer` object with ambient credentials, you must enable the `--issuer-ambient-credentials` setting for the cert-manager controller.
====
```